workers/udp.go

```diff
@@ -0,0 +1,130 @@
+package workers
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/frightenedmonkey/statsproxy/common"
+	"github.com/frightenedmonkey/statsproxy/config"
+	"net"
+	"strconv"
+	"strings"
+	"sync/atomic"
+)
+
+type Packet struct {
+	Bucket string
+	Raw    string
+}
+
+func StartUDPListeners(l *net.UDPConn) {
+	for i := 0; i < config.Service.Workers; i++ {
+		go udpListener(l)
+	}
+}
+
+func StartUDPWriters() {
+	for i := 0; i < config.Service.WriterMultiplier; i++ {
+		for _, host := range config.Service.Statsd.Hosts {
+			go hostWriter(host)
+		}
+	}
+}
+
+func hostWriter(host config.Host) {
+	coalescedPacket := make([]byte, 0)
+	for {
+		newMessage := <-common.HostChans[host]
```

Comment:
Well, it's in a goroutine, so presumably it'd just sit there and do nothing. With that said, the `HostChans` map is initialized with all the same hosts here: https://github.com/frightenedmonkey/statsproxy/pull/4/files#diff-274f48f22fbd52b3aa5b2df88aa224e0R33